lib/srv/termmanager.go

```diff
@@ -99,21 +99,37 @@ func (g *TermManager) writeToClients(p []byte) {
 	g.history = truncateFront(append(g.history, p...), maxHistoryBytes)
 
 	atomic.AddUint64(&g.countWritten, uint64(len(p)))
+	var toDelete []struct {
+		key string
+		err error
+	}
```

Comment:
Just for future reference, you can do something like this:

```go
	type deleteKey struct {
		key string
		err error
	}
	var toDelete []deleteKey
```

so you don't have to keep re-defining the struct everywhere.